source/includes/aggregation.scala

```diff
@@ -0,0 +1,41 @@
+import org.mongodb.scala._
+import org.mongodb.scala.model.{ Aggregates, Filters, Accumulators }
+import org.mongodb.scala.bson.Document
+import com.mongodb.ExplainVerbosity
+
+object Aggregation {
+
+  def main(args: Array[String]): Unit = {
+    val mongoClient = MongoClient("<connection string URI>")
+
+    val database: MongoDatabase = mongoClient.getDatabase("sample_restaurants")
+    val collection: MongoCollection[Document] = database.getCollection("restaurants")
+
+    // Retrieves documents with a cuisine value of "Bakery", groups them by "borough", and
+    // counts each borough's matching documents
+    // start-match-group
+    val pipeline = Seq(Aggregates.filter(Filters.equal("cuisine", "Bakery")),
+                       Aggregates.group("$borough", Accumulators.sum("count", 1))
+    )
+
+    collection.aggregate(pipeline)
+              .subscribe((doc: Document) => println(doc.toJson()),
+                        (e: Throwable) => println(s"There was an error: $e"))
+    // end-match-group
+
+    // Performs the same aggregation operation as above but asks MongoDB to explain it
+    // start-explain
+    val pipeline = Seq(Aggregates.filter(Filters.equal("cuisine", "Bakery")),
```

Comment:
I see why you re-use the `pipeline` variable name here, but it makes it so that you can't actually compile/run this file.  I suggest either splitting this into two Scala classes, or using a different variable name here, e.g. `pipelineToExplain`